example/host_translations.csv

```diff
@@ -0,0 +1,55 @@
+ipaddr,hostname
```

Comment:
For the convenience of quickly knowing which dbhost is having issues.  Otherwise, no need.